src/components/SidebarMain/SidebarMain.js

```diff
@@ -27,6 +27,17 @@ const menuItemsBeheer = {
     Beleidsprestaties: { url: '/muteer/beleidsprestaties', finished: true },
 }
 
+/**
+ * Component that renders the ReturnNavLink component, which displays the value parameter with an arrow in which a user can navigate on click.
+ *
+ * @component
+ *
+ * @param {string} value - Parameter containing a value.
```

Comment:
Parameter containing a value. -> Contains the text to display

Reply:
I've changed the parameter description.